internal/server/stat/stat_reader.go

```diff
@@ -211,6 +211,10 @@ func ReadStatsPb(
 			)
 			result[place][sv].PlaceName = obs.PlaceName
 		}
+		// Always prefer data from branch cache.
+		if len(btData) > 0 && btGroup.TableNames()[i] == btGroup.BranchTableName() {
```

Comment:
To confirm, should we do this per SV / place?  Because what if we fix a certain SV but not others?

Reply:
Updated